R/utils-format.r

```diff
@@ -298,6 +298,17 @@ pluralise <- function(message, objects) {
   message
 }
 
+mult_sign <- function(with_spaces = TRUE) {
+  # unicode multiplication sign
+  mult <- "\u00d7"
+  # if unicode doesn't render, use lowercase x
+  if(enc2native(mult) != mult) mult <- "x"
```

Comment:
Thanks. Could you please match the code style to the surrounding code (space after `if`, braces)?